.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "flatpak-builder-tools"]
+	path = flatpak-builder-tools
+	url = https://github.com/nanonyme/flatpak-builder-tools.git
+	branch = support-build-isolation
```

Comment:
These two lines should be updated to reflect the true `flatpak-builder-tools` repository:

```
url = https://github.com/flatpak/flatpak-builder-tools.git
branch = master
```